### PR TITLE
Add ianvs command line usage

### PR DESCRIPTION
**Pull request by JimmyYang20**

### CODE REVIEW

File: docs/user_inferface/how-to-use-ianvs-command-line.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# How to use Ianvs command line
+
+## Ianvs
```
Comment (MooreZheng) [inline docs/user_inferface/how-to-use-ianvs-command-line.md]: It is suggested to list diffrent commands independently as sections

File: docs/user_inferface/how-to-use-ianvs-command-line.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+To list available commands, either run ianvs with no parameters or execute ianvs help:
```
Comment (MooreZheng) [inline docs/user_inferface/how-to-use-ianvs-command-line.md]: For each command, list what action is taken and its output result

File: docs/user_inferface/how-to-use-ianvs-command-line.md (hunk 1 above)
Comment (JimmyYang20) [reply docs/user_inferface/how-to-use-ianvs-command-line.md]: fix it

**Pull request merged**